sdk/python/feast/pyarrow_type_map.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2019 The Feast Authors
```

Comment:
Does this have to be a new file? What makes pyarrow so special?

Reply:
Yes, it can be part of `type_map.py`, I'll move it back into `type_map.py`.

Reply:
Thanks

Reply:
fixed